app/services/github_api.rb

```diff
@@ -0,0 +1,41 @@
+require 'httparty'
+require 'json'
+require 'pry'
+require './app/poros/repo'
+require './app/poros/user_info'
+require './app/poros/commit'
+require './app/poros/pull_request'
+
+response = HTTParty.get("https://api.github.com/repos/Sierra-T-9598/little-esty-shop")
+parsed = JSON.parse(response.body, symbolize_names: true)
+
+repo = Repo.new(parsed[:name])
+
+#####
+
+user_response = HTTParty.get("https://api.github.com/repos/Sierra-T-9598/little-esty-shop/contributors")
+user_parsed = JSON.parse(user_response.body, symbolize_names: true)
+
+users = user_parsed.map do |user_data|
+  UserInfo.new(user_data)
+end
+
+#####
+
```

Comment:
https://stackoverflow.com/questions/30656761/github-search-api-only-return-30-results 
I think there is a way to get even more than 100 back

Reply:
Ah, we could totally try that. Just plug in a `?page=2` on the end of the query and see if we can't increase the number of commits we are allowed to view.